GPU/GPUTracking/Base/hip/CMakeLists.txt

```diff
@@ -51,7 +51,7 @@ if(ALIGPU_BUILD_TYPE STREQUAL "O2")
 
   install(FILES ${HDRS} DESTINATION include/GPU)
 
- if(NOT $ENV{ALIBUILT_O2_TESTS})
+ if(ENABLE_HIP AND NOT ENV{ALIBUILD_O2_TESTS})
```

Comment:
after  discussion with @ktf we decided that this is not the right approach. Actually, we want to build the tests if we have CUDA and HIP available but exclude them from running when no cards are present, which can be done in the O2 recipe.

Reply:
makes sense for me